src/orion/core/worker/experiment.py

```diff
@@ -190,20 +199,12 @@ def __getstate__(self):
         for entry in self.__slots__:
             state[entry] = getattr(self, entry)
 
-        # TODO: This should be removed when singletons and `get_storage()` are removed.
```

Comment:
:tada: :tada: :tada: 

Sorry for spamming. This change makes me very happy. :smile: